homeworks/vadym.myroshnyk_VadimMir/homework_7/homework_7.js

```diff
@@ -0,0 +1,19 @@
+// fibonacci
+const randomNum = Math.floor(Math.random() * (100 - 50 + 1) + 50);
+
+function fib(random) {
+    const arr = [0];
+    let a = 1;
+    let b = 0;
+    for (let i = 0; i <= random; i += a) {
```

Comment:
`i <= random`? and what if this condition is met, but `b` is already `> random`? you will push a number that is already bigger than max allowed value

with this line of code it's possible